pkg/parse/parse.go

```diff
@@ -64,29 +64,20 @@ func Parse() {
 		os.Exit(1)
 	}
 
-	currBranch := o.RepoHead.Value()
-
-	cmd := git.Git{Workspace: o.Dir.Value(), Head: currBranch, RepoName: o.RepoName.Value()}
-
-	// TODO: Reintroduce this codepath if we decide the flag finder should be able to clone repos
-	// endpoint := o.CloneEndpoint.Value()
-	// if endpoint != "" {
-	// 	dir, err := ioutil.TempDir("", cmd.RepoName)
-	// 	if err != nil {
-	// 		fatal("Failed to create temp directory for repo clone", err)
-	// 	}
-	// 	defer os.RemoveAll(dir)
-	// 	cmd.Workspace = dir
-	// 	err = cmd.Clone(endpoint)
-	// 	if err != nil {
-	// 		fatal("Unable to clone repo", err)
-	// 	}
-	// }
```

Comment:
Decided it's pretty safe to remove this commented out code path. If we ever want this functionality back, we can do a git revert on the commit that commented this code out.